### ISSUE TITLE
Switch to matrices for NBCs for non-overlap and Schwarz contacts

### ISSUE THREAD

**Issue by lxmota** [sandialabs/Norma.jl#29]
The tractions are returned in an array that has to be indexed in an awkward manner instead of 3 x number-nodes as all other quantities. Change that to be consistent.

**Comment by ikalash** [comment]
So this is the issue that was messing up the non-overlap coupling?  Thanks for finding it!  How come it worked for contact for Daria's test cases?

**Comment by lxmota** [comment]
No, not really. The issue is that I think we should do for non-overlap coupling the same things that we are doing for contact while in the contact phase. I ran the cuboids problem with contact and it works, so there is a difference or differences in what we are doing for coupling but I haven't discovered what those are yet.

**Comment by lxmota** [comment]
Fixed a long time ago.
